php/formatting-functions.php

```diff
@@ -35,3 +35,30 @@ function wp_seo_no_formatting_tags_allowed( $string ) {
 
 	return $string;
 }
+
+/**
+ * Get the character count of a format string for display to users.
+ *
+ * @since 0.13.0
+ *
+ * @param  string $string String to count.
+ * @return string         The character count as a string, or a description of
+ *                        the count if it can't be calculated precicsely.
```

Comment:
* sp: precisely
* Can you add a note explaining why the count is returned as a string?